test/runtests.jl

```diff
@@ -85,6 +85,11 @@ const colon = Base.:(:)
     @test ConstructionBase.constructorof(typeof(1.0m))(2) === 2m
 end
 
+VERSION >= v"1.9-" && @testset "inverse" begin
+    InverseFunctions.test_inverse(Base.Fix1(ustrip, u"m"), 2u"m")
+    InverseFunctions.test_inverse(Base.Fix1(ustrip, u"m"), 2u"mm")
```

Comment:
```suggestion
    InverseFunctions.test_inverse(Base.Fix1(ustrip, m), 2m)
    InverseFunctions.test_inverse(Base.Fix1(ustrip, m), 2mm)
```
The units `m` and `mm` are imported in the test suite, so this can be simplified.